_topic_map.yml

```diff
@@ -754,21 +754,23 @@ Topics:
   File: setting-up-trusted-ca
   Distros: openshift-enterprise,openshift-webscale,openshift-origin,openshift-dedicated
 ---
-#Name: OpenShift Pipelines
-#Dir: openshift_pipelines
-#Distros: openshift-enterprise,openshift-webscale,openshift-origin
-#Topics:
-#- Name: Understanding OpenShift Pipelines
-#  File: understanding-openshift-pipelines
-#- Name: Installing Pipelines
-#  File: installing-pipelines
-#- Name: Uninstalling Pipelines
-#  File: uninstalling-pipelines
-#- Name: Getting started with OpenShift Pipelines
-#  File: getting-started-with-openshift-pipelines
-#- Name: Working with Pipelines using the Developer perspective
-#  File: working-with-pipelines-using-the-developer-perspective
-#---
+Name: Pipelines
+Dir: pipelines
```

Comment:
I am not sure if you can change the directory name. It has to be changes in the system as well, else the URLs will fail.

Reply:
Retaining it as it was earlier.